Casks/a/atv-remote.rb

```diff
@@ -0,0 +1,22 @@
+cask "atv-remote" do
+  arch arm: "-arm64"
+
+  version "1.3.7"
+  sha256 arm:   "9d38b971f15fcb4408cdfaea6e4c3c2885939f73b2fc0249aa6e9d15150bd10a",
+         intel: "1956ca8b1092e08296813335f147573925331e0b2ed6b8cf7b72f80f58e7ff95"
+
+  url "https://github.com/bsharper/atv-desktop-remote/releases/download/v.#{version}/ATV.Remote-#{version}#{arch}.dmg"
+  name "ATV Remote"
+  desc "Control Apple TV from your desktop"
+  homepage "https://github.com/bsharper/atv-desktop-remote"
+
+  depends_on macos: ">= :high_sierra"
```

Comment:
```suggestion
  depends_on formula: "python@3.13"
  depends_on macos: ">= :high_sierra"
```

The app seems to require Python.

Reply:
Upstream seems to prefer the system Python (3.9) installed as part of the CLT, even though I only have 3.13 installed and it is the preferred Python in $PATH.

In a clean VM with only Homebrew installed, the app works fine with Python 3.9.

```
…/Library/Application Support/ATV Remote/env > cat pyvenv.cfg
home = /Library/Developer/CommandLineTools/usr/bin
include-system-site-packages = false
version = 3.9.6
```

```
> which python3
/opt/homebrew/bin/python3
> python3 --version
Python 3.13.2
```

Reply:
Great, one less dependency to worry about. Thanks for confirming!